src/Categories/Diagram/Duality.agda

```diff
@@ -179,3 +180,50 @@ module _ {F : Bifunctor (Category.op D) D C} where
     ; unique    = unique
     }
     where open Coend.Coend e
+
+
+module DualityConversionProperties where
+  private
+    Coequalizer⇔coEqualizer : ∀ (coequalizer : Coequalizer f g) →
+      coEqualizer⇒Coequalizer (Coequalizer⇒coEqualizer coequalizer)≡ coequalizer
```

Comment:
can you add a space after the `(` ?  I had to read this twice to 'get' the type.

Reply:
done